.github/workflows/preview_ci.yml

```diff
@@ -77,6 +77,7 @@ jobs:
         rm -rf ./src/OrchardCore.Cms.Web/App_Data
         rm -rf ./src/OrchardCore.Cms.Web/App_Data_Tests
         dotnet publish -c Release -o ./.build/release --no-build --no-restore --framework net6.0
+        cp -R ./src/OrchardCore.Cms.Web/Localization ./.build/release/Localization
```

Comment:
Do we still need this line then?

Normally with the new little target it populates the translation files under the publish output path

Idem below

Reply:
Yes if we want to fix the current 1.1.0 Docker image we need to fix it from the CI. It can be removed afterward.

Reply:
Now the other question is: is this necessary that we keep this part?

https://github.com/OrchardCMS/OrchardCore/blob/aa724a3a56c71f4d32b1973e4a2e73255b90d5cf/src/OrchardCore/OrchardCore.Application.Cms.Core.Targets/OrchardCore.Application.Cms.Core.Targets.targets#L36-L41

Reply:
Removed it as it's kind of doing it afterward anyway.

Reply:
Yes it looks like, I assume that you tried it ;)

About not modifying the _ci.yml

> Yes if we want to fix the current 1.1.0 Docker image we need to fix it from the CI. It can be removed afterward.

Could we just do the same (only the suggested change in the .targets) in separate PRs targetting the version branches that need to be updated?

Reply:
I think we use tags for releases